### PR TITLE
karate freestyle 5” tune

### PR DESCRIPTION
**Pull request by sugaarK**
This is the sugarK freestyle tune, its been test over the last few months by members of the UAVtech discord.. This is a starting point tune and has siders active so you can fine tune it easily.

### CODE REVIEW

File: presets/4.3/tune/karate_freestle.txt [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+# -- Dterm filtering --
+set dterm_lpf1_dyn_expo = 10
+
+# -- RPM filtering --
+set dshot_bidir = ON
+set rpm_filter_fade_range_hz = 100
+
+# -- Misc --
+set yaw_spin_recovery = AUTO
+set thrust_linear = 0
```
Comment (limonspb) [inline presets/4.3/tune/karate_freestle.txt:43]: thrust_linear  = 0 and yaw_spin_recovery = AUTO are like that in TUNE defaults anyways.

File: presets/4.3/tune/karate_freestle.txt [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+set d_min_roll = 43
+set d_min_pitch = 47
+set simplified_pids_mode = RP
+set simplified_d_gain = 145
+set simplified_pi_gain = 120
+set simplified_dmax_gain = 70
+set simplified_feedforward_gain = 115
+set simplified_pitch_d_gain = 110
+set simplified_pitch_pi_gain = 110
+set simplified_dterm_filter = ON
```
Comment (limonspb) [inline presets/4.3/tune/karate_freestle.txt:63]: this should go to D term filters section i  guess for more nice readings

File: presets/4.3/tune/karate_freestle.txt [hunk 3]
```diff
@@ -0,0 +1,87 @@
...
+set simplified_dterm_filter_multiplier = 100
+
+# -- Feedforward --
+set feedforward_averaging = 2_POINT
+set feedforward_jitter_reduction = 12
+
+#$ OPTION BEGIN (UNCHECKED): Dshot300
+set dshot_bidir = ON
```
Comment (limonspb) [inline presets/4.3/tune/karate_freestle.txt:71]: dshot_bidir not needed here as its in the main section

File: presets/4.3/tune/karate_freestle.txt [hunk 4]
```diff
@@ -0,0 +1,87 @@
...
+set motor_pwm_protocol = Dshot300
+#$ OPTION END
+
+#$ OPTION BEGIN (UNCHECKED): Dshot600
+set dshot_bidir = ON
```
Comment (limonspb) [inline presets/4.3/tune/karate_freestle.txt:76]: dshot_bidir not needed here as its in the main section

File: presets/4.3/tune/karate_freestle.txt [hunk 5]
```diff
@@ -0,0 +1,87 @@
...
+set motor_pwm_protocol = DSHOT600
+#$ OPTION END
+
+#$ OPTION BEGIN (UNCHECKED): dynamic idle
```
Comment (limonspb) [inline presets/4.3/tune/karate_freestle.txt:80]: are you sure you want it UNchecked? just checking

**Review by limonspb — CHANGES_REQUESTED** [review]

File: presets/4.3/tune/karate_freestle.txt [hunk 6]
```diff
@@ -0,0 +1,87 @@
+Karate FS tune
+#$ TITLE: Karate Freestyle
```
Comment (limonspb) [inline presets/4.3/tune/karate_freestle.txt:2]: Maybe add "Karate Freestyle **5''**" for the title.
I figured we need to specify more which this or that tune is suitable for.

**Review by limonspb — CHANGES_REQUESTED** [review]

File: presets/4.3/tune/karate_freestle.txt (hunk 6 above)
Comment (sugaarK) [reply presets/4.3/tune/karate_freestle.txt:2]: Easy sorry for the mistakes 😂 I’ll fix it up when I get home

File: presets/4.3/tune/karate_freestle.txt (hunk 1 above)
Comment (sugaarK) [reply presets/4.3/tune/karate_freestle.txt:43]: good point

File: presets/4.3/tune/karate_freestle.txt (hunk 3 above)
Comment (sugaarK) [reply presets/4.3/tune/karate_freestle.txt:71]: fixed

File: presets/4.3/tune/karate_freestle.txt (hunk 4 above)
Comment (sugaarK) [reply presets/4.3/tune/karate_freestle.txt:76]: fixed

File: presets/4.3/tune/karate_freestle.txt (hunk 6 above)
Comment (sugaarK) [reply presets/4.3/tune/karate_freestle.txt:2]: good point

**Review by limonspb — APPROVED** [review]

**Comment by Canaill51** [comment]
Hi! Just a quick question regarding to this preset:
Is it recommended with ESC PWM 24kHz or 48kHz?

I did try this preset on two different machines (4s & 6s), running on 24kHz ESC PWM. No issue, fly great. Very impressed about handling and propwash performance. 

Just to get your confirmation (maybe great to add this ESC requirement into this preset?).

Thanks again for your support.

**Comment by sugaarK** [comment]
> Hi! Just a quick question regarding to this preset: Is it recommended with ESC PWM 24kHz or 48kHz?
> 
> I did try this preset on two different machines (4s & 6s), running on 24kHz ESC PWM. No issue, fly great. Very impressed about handling and propwash performance.
> 
> Just to get your confirmation (maybe great to add this ESC requirement into this preset?).
> 
> Thanks again for your support.

Imo the lowest pwm rate you can get away with is the best one. If you ur using blhlie32 stuff then 32.9 with 24-rpm would be the winner. On bluejay I’d use what flys the best. Unfortunately pwm rates are a bit of a black art and only trying will give you the proper answers for your setup

**Comment by Chekagos** [comment]
I spent two weeks to configure my new assembly BF4.4.2   FC-H7   QAV JB2 frame.
There were problems in this configuration.
The experience of previous BF tune versions did not help me. (crazy wobbling and unstable at all)
I am very grateful for this preset.
Thank you very much for the long and hard work.
This preset deserves - gold in 2023.

**Pull request merged**